code/game/machinery/doors/airlock.dm

```diff
@@ -876,6 +876,34 @@ About the new airlock wires panel:
 	return 1
 
 
+
+/obj/machinery/door/airlock/proc/CreateAssembly()
+	var/obj/structure/door_assembly/da = new assembly_type(src.loc)
+	if (istype(da, /obj/structure/door_assembly/multi_tile))
+		da.set_dir(src.dir)
+
+	da.anchored = 1
+	if(mineral)
+		da.glass = mineral
+	//else if(glass)

```

Comment:
Minor Code-Quality nitpick.
Serves no purpose and is commented out -> Remove please